src/pages/Environments.js

```diff
@@ -241,22 +250,33 @@ const Environments = ({ namespace }) => {
     }, [h(Link, ['details'])])
   }
 
```

Comment:
`renderDeleteButton` replaces `renderDeleteButtonApps` and `renderDeleteButtonRuntimes` to reduce code duplication.